query_execution/QueryExecutionTypedefs.hpp

```diff
@@ -66,6 +67,19 @@ enum QueryExecutionMessageType : message_type_id {
   kRebuildWorkOrderMessage,  // From Foreman to Worker.
   kRebuildWorkOrderCompleteMessage,  // From Worker to Foreman.
   kPoisonMessage,  // From the CLI shell to Foreman, then from Foreman to Workers.
+
```

Comment:
Would you consider sorting this list to improve readability? 

I know the previous enum was also not sorted and probably should be too.